src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -0,0 +1,852 @@
+            optional=false)
+    private List<String> USE_ANNOTATIONS = new ArrayList<>();
+
+    /**
+     * Add truth sensitivity slices through the call set at the given values. The default values are 100.0, 99.9, 99.0, and 90.0
+     * which will result in 4 estimated tranches in the final call set: the full set of calls (100% sensitivity at the accessible
+     * sites in the truth set), a 99.9% truth sensitivity tranche, along with progressively smaller tranches at 99% and 90%.
+     */
+    @Argument(fullName="TStranche",
+            shortName="tranche",
+            doc="The levels of truth sensitivity at which to slice the data. (in percent, that is 1.0 for 1 percent)",
+            optional=true)
+    private List<Double> TS_TRANCHES = new ArrayList<Double>(Arrays.asList(100.0, 99.9, 99.0, 90.0));
+
+    /**
+     * For this to work properly, the -ignoreFilter argument should also be applied to the ApplyRecalibration command.
+     */
+    @Argument(fullName="ignore_filter",
+            shortName="ignoreFilter",
+            doc="If specified, the variant recalibrator will also use variants marked as filtered by the specified filter name in the input VCF file",
+            optional=true)
+    private List<String> IGNORE_INPUT_FILTERS = new ArrayList<>();
+
+    @Argument(fullName="ignore_all_filters",
+            shortName="ignoreAllFilters",
+            doc="If specified, the variant recalibrator will ignore all input filters. Useful to rerun the VQSR from a filtered output file.",
+            optional=true)
+    private boolean IGNORE_ALL_FILTERS = false;
+
+    @Argument(fullName="rscriptFile", shortName="rscriptFile", doc="The output rscript file generated by the VQSR to aid in visualization of the input data and learned model", optional=true)
+    private String RSCRIPT_FILE = null;
+
+    /**
+     *  This GATKReport gives information to describe the VQSR model fit. Normalized means for the positive model are
+     *  concatenated as one table and negative model normalized means as another table. Covariances are also concatenated
+     *  for positive and negative models, respectively. Tables of annotation means and standard deviations are provided
+     *  to help describe the normalization. The model fit report can be read in with our R gsalib package. Individual
+     *  model Gaussians can be subset by the value in the "Gaussian" column if desired.
+     */
+    @Argument(fullName="output_model",
+            shortName = "outputModel",
+            doc="If specified, the variant recalibrator will output the VQSR model fit to the file specified by -modelFile or to stdout",
+            optional=true)
+    private boolean outputModel = false;
+
+    @Argument(fullName="model_file",
+            shortName = "modelFile",
+            doc="A GATKReport containing the positive and negative model fits",
+            optional=true)
+    private String modelReport = null;
+
+    @HiddenOption
```

Comment:
We have both `@Hidden` and `@HiddenOption` -- is either one actually hooked up to the arg system?